web/profiles/joinup/joinup.info.yml

```diff
@@ -39,7 +40,6 @@ dependencies:
   - rdf_entity
   - rdf_export
   - rdf_taxonomy
-  - solution
   - state_machine
```

Comment:
:( clumsy as hell.. Well, it seems that I had not set the dependencies for the joinup_news module properly so it did not install everything properly.. I reverted the sorting.